microprofile-jwt-auth-tool/src/main/java/org/jboss/eap/qe/microprofile/jwt/auth/tool/JwtHelper.java

```diff
@@ -0,0 +1,63 @@
+package org.jboss.eap.qe.microprofile.jwt.auth.tool;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.Signature;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.UUID;
+
+/**
+ * Sugar syntax for generating JWTs with re-used values
+ */
+public final class JwtHelper {
+    private final RsaKeyTool keyTool;
+    private final String issuer;
+
+    public JwtHelper(final RsaKeyTool keyTool, final String issuer) {
+        this.keyTool = keyTool;
+        this.issuer = issuer;
+    }
+
+    public JsonWebToken generateProperSignedJwt() {
```

Comment:
'Proper' - what does it stand for ?

why not just `generateSignedJwt`

Reply:
Proper because it has all values set according to specification. I will add the JavaDoc here.

Reply:
My point why it needs to be highlighted in the method name that it does what it should do ...

+1 for JavaDoc